File: python3/fizzbuzz.py
def fizz_buzz():
    output_values = []

    for i in range(1, 101):
        fizzbuzz_string = ''

        if (i % 3 == 0):
            fizzbuzz_string += 'Fizz'
        if (i % 5 == 0):
            fizzbuzz_string += 'Buzz'

        output_values.append(fizzbuzz_string if len(fizzbuzz_string) else i)

    output_string = ' '.join(str(value) for value in output_values)
    print(output_string)

fizz_buzz()

File: javascript/fizzbuzz.js
(function fizzBuzz() {
    var i;
    var fizzBuzzString;
    var outputValues = [];

    for (i = 1; i <= 100; i++) {
        fizzBuzzString = '';

        if (i % 3 === 0) {
            fizzBuzzString += 'Fizz';
        }
        if (i % 5 === 0) {
            fizzBuzzString += 'Buzz';
        }

        outputValues.push(fizzBuzzString.length ? fizzBuzzString : i);
    }

    console.log(outputValues.join(' '));
})();
